### PR TITLE
SLCORE-438 map securityStandard to vulnerability probability

### PR DESCRIPTION
**Pull request by sophio-japharidze-sonarsource**

### CODE REVIEW

File: core/src/test/java/org/sonarsource/sonarlint/core/tracking/ServerHotspotTrackableTests.java [hunk 1]
```diff
@@ -24,6 +24,7 @@
 import org.sonarsource.sonarlint.core.commons.RuleType;
 import org.sonarsource.sonarlint.core.commons.TextRange;
 import org.sonarsource.sonarlint.core.serverapi.hotspot.ServerHotspot;
+import org.sonarsource.sonarlint.core.commons.VulnerabilityProbability;
```
Comment (henryju) [inline core/src/test/java/org/sonarsource/sonarlint/core/tracking/ServerHotspotTrackableTests.java:27]: Useless import?

File: rule-extractor/pom.xml [hunk 2]
```diff
@@ -59,6 +60,10 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```
Comment (henryju) [inline rule-extractor/pom.xml:63]: Why do you need this dependency? If you really need it, please put it with other production dependencies, not with test dependencies.

**Review by henryju — CHANGES_REQUESTED** [review]
A few comments, but overall LGTM!

File: core/src/test/java/org/sonarsource/sonarlint/core/tracking/ServerHotspotTrackableTests.java (hunk 1 above)
Comment (sophio-japharidze-sonarsource) [reply core/src/test/java/org/sonarsource/sonarlint/core/tracking/ServerHotspotTrackableTests.java:27]: Needed for line 36

File: rule-extractor/pom.xml (hunk 2 above)
Comment (sophio-japharidze-sonarsource) [reply rule-extractor/pom.xml:63]: It was needed for the new `SecurityStandards` class, to use `ImmutableMap` and `Ordering.` Moved it under regular dependencies

Comment (henryju) [reply rule-extractor/pom.xml:63]: Normally, with Java 11+, you can avoid using Guava, it will allow to keep SLCORE distribution smaller

**Review by henryju — APPROVED** [review]

**Pull request merged**